static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -204,6 +232,52 @@ export class AccessMethod extends Component {
     })
   }
 
+  handleHarmonySelection(event, harmonyMethods) {
+    if (event !== '') {
+      this.setState({
+        selectValue: harmonyMethods.find(({ methodKey }) => methodKey === event).name
+      })
+
+      this.handleAccessMethodSelection(event)
+    }
+  }
+
+  harmonyMethodsMapper(methods, selected) {
+    return (
+      <div id="harmony_methods">
+        {methods.map((radio) => this.renderHarmonySelectItem(radio, selected))}
```

Comment:
```suggestion
        {methods.map((radio) => this.createHarmonySelectItem(radio, selected))}
```
Just nitpick I'd like to just obfuscate the word render from this